### PR TITLE
ACM-8264--fix known issues for gh file (MOVED and CLOSED)

### PR DESCRIPTION
**Pull request by swopebe**
…le and add new

### CODE REVIEW

File: global_hub/global_hub_known_issues.adoc [hunk 1]
```diff
@@ -24,7 +24,26 @@ Or consider a troubleshooting topic.
 Review the known issues for multcluster global hub. The following list contains known issues for this release, or known issues that continued from the previous release. For your {ocp-short} cluster, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14[{ocp-short} release notes].
 
 
-[#global-hub-issues]
-== Multicluster global hub
+[#managed-cluster-not-counted]
+== Managed cluster displays but is not counted
```
Comment (swopebe) [inline global_hub/global_hub_known_issues.adoc:29]: We already titled the file this; we don't need to repeat the titled on line 27. This is where we list the known issues, same as all the other files.

File: global_hub/global_hub_known_issues.adoc [hunk 2]
```diff
@@ -24,7 +24,26 @@ Or consider a troubleshooting topic.
...
+
+
+[#no-new-page-group-filter]
+== Globalhub Operator installed in {ocp-short} 4.13 might redirect
+
+In the Global Hub Policy Group Compliancy Overview hub dashboards, you can check one data point by clicking **View Offending Policies for standard group**, but after you click this link to go to the offending page, the standard group filter cannot pass to the new page. 
+
+This is also an issue for the **Cluster Group Compliancy Overview**.
+
+//workaround?
```
Comment (swopebe) [inline global_hub/global_hub_known_issues.adoc]: need to 1. Check to see if this is the best strategy long term
2. ask for workarounds
3. Make sure naming is proper. @cadawson have you checked the build for UI control and such and can you point me to that build? TY!

File: global_hub/global_hub_known_issues.adoc (hunk 1 above)
Comment (swopebe) [reply global_hub/global_hub_known_issues.adoc:29]: We will copy from here before deleting this file.

File: global_hub/global_hub_known_issues.adoc (hunk 2 above)
Comment (swopebe) [reply global_hub/global_hub_known_issues.adoc]: @ldpliu do you mind helping me answer the questions? Are there any work arounds to these and are these the names of the components in the UI?

Strategy has been answered by PM, the file is moving to RHACM release notes after we are done.

**Comment by ldpliu** [comment]
As we still need to add 3 new known issues in [this pr ](https://github.com/stolostron/multicluster-global-hub/pull/706)
So I sumarized **all** the known issues(include old and new contexts) for global hub 1.0 release (include workaround if it have). Please update the known issues based on this draft.

```
## Managed cluster displays but is not counted

A managed cluster that is not created successfully, meaning clusterclaim id.k8s.io` does not exist in the managed cluster, is not counted in global hub policy compliance dashboards, but it shows in the policy console.

## Globalhub Operator installed in {ocp-short} 4.13 might redirect

If the Globalhub Operator is installed in {ocp-short} 4.13, all hyperlinks that link to the managed clusters list and detail pages in dashboards might redirect to the {product-title-short} home page.

Please manually redirect to you target page.

## The standard group filter cannot pass to the new page

In the Global Hub Policy Group Compliancy Overview hub dashboards, you can check one data point by clicking `View Offending Policies for standard group`, but after you click this link to go to the offending page, the standard group filter cannot pass to the new page.

This is also an issue for the Cluster Group Compliancy Overview.

Please manually select the target filters.

## Can not view old managed hub data in some dashboard when reimport a managed hub cluster with different name
When reimport a managed hub cluster with different name. The old managed hub data will be null in the following dashboards. `Global Hub - Ad Hoc Policy Investigation` `Global Hub - What's Changed / Clusters` and `Global Hub - What's Changed / Policies`.
So when you import a managed hub cluster, it's better to use the same name.

## Detach Managed Hub may cause applications and policies deleted.
If Global Hub running on RHACM 2.7.9, When the Managed Hub is detached, all policies and applications created on managed hub will be deleted.

## Policy cannot be propagated to the Managed Hub cluster
If Global Hub is running on RHACM 2.8.3 or below, policy cannot be propagated to the managed hub cluster due to https://issues.redhat.com/browse/ACM-8338. 
The workaround is to create a namespace <Managed Hub Name> in the managed hub cluster.

## Can not redirect to OpenShift 3.11 cluster Observability page.
If a managed hub import an Openshift 3.11 cluster as managed cluster, It can not redirect to the `RHACM Observability` page in `Global Hub - Overview` dashboard.

Please manually redirect to your target page.
```

File: global_hub/global_hub_known_issues.adoc [hunk 3]
```diff
@@ -24,7 +24,26 @@ Or consider a troubleshooting topic.
...
+[#operator-hyperlink]
+== Globalhub Operator installed in {ocp-short} 4.13 might redirect
+
+If the Globalhub Operator is installed in {ocp-short} 4.13, all hyperlinks that link to the managed clusters list and detail pages in dashboards might redirect to the {product-title-short} home page. 
+
+//workaround?
+
+
+[#no-new-page-group-filter]
+== Globalhub Operator installed in {ocp-short} 4.13 might redirect
```
Comment (clyang82) [inline global_hub/global_hub_known_issues.adoc]: the title is the same as above. @ldpliu ?

File: global_hub/global_hub_known_issues.adoc (hunk 2 above)
Comment (ldpliu) [reply global_hub/global_hub_known_issues.adoc]: > @ldpliu do you mind helping me answer the questions? Are there any work arounds to these and are these the names of the components in the UI?
> 
> Strategy has been answered by PM, the file is moving to RHACM release notes after we are done.

Please use this draft.
https://github.com/stolostron/rhacm-docs/pull/5729#issuecomment-1791986925

File: global_hub/global_hub_known_issues.adoc (hunk 3 above)
Comment (ldpliu) [reply global_hub/global_hub_known_issues.adoc]: > the title is the same as above. @ldpliu ?

Changed in https://github.com/stolostron/rhacm-docs/pull/5729#issuecomment-1791986925

File: global_hub/global_hub_known_issues.adoc [hunk 4]
```diff
@@ -21,10 +21,43 @@ Known issue process and when to write:
...
+
+[#detach-managed-hub-deletions]
+== Detach managed hub cluster causes applications and policies deletions
+
+If Multicluster Global Hub Operator is running on {product-title-short} 2.7.9 when the managed hub cluster is detached, all policies and applications that are created on the managed hub cluster are deleted.
+
+[#policy-cannot-be-propagated-hub]
+== Policy cannot be propagated to the managed hub cluster
+
+If Multicluster Global Hub Operator is running on {product-title-short} 2.8.3 or earlier, policy cannot be propagated to the managed hub cluster due to https://issues.redhat.com/browse/ACM-8338. (not this)
```
Comment (swopebe) [inline global_hub/global_hub_known_issues.adoc]: line 54 - 56, please change. We don't point to the issues here and I didn't glean much from the issue anyway.

We need the cause and I don't think line 56 tells me much either.

**Comment by swopebe** [comment]
@ldpliu please see my comment and changes.

File: global_hub/global_hub_known_issues.adoc (hunk 4 above)
Comment (swopebe) [reply global_hub/global_hub_known_issues.adoc]: Assigning @cadawson for review

Comment (ldpliu) [reply global_hub/global_hub_known_issues.adoc]: @swopebe  Please change it to:
`If Multicluster Global Hub Operator is running on {product-title-short} 2.8.3 or earlier, policy cannot be propagated to the managed hub cluster due to the cluster namespace is not exist.`

Comment (swopebe) [reply global_hub/global_hub_known_issues.adoc]: @ldpliu changed! Thank you so much.

Can you approve if there is nothing else that needs to change?

File: global_hub/global_hub_known_issues.adoc [hunk 5]
```diff
@@ -21,10 +21,43 @@ Known issue process and when to write:
...
+
+[#operator-hyperlink]
+== Global Hub Operator installed on {ocp-short} 4.13 hyperlinks might redirect home
+
+If the Multicluster Global Hub Operator is installed on {ocp-short} 4.13, all hyperlinks that link to the managed clusters list and detail pages in dashboards might redirect to the {product-title-short} home page. 
+
+You need to manually go to your target page.
+
+[#no-new-page-group-filter]
+== Multicluster Global Hub Operator installed in {ocp-short} 4.13 might redirect
```
Comment (ldpliu) [inline global_hub/global_hub_known_issues.adoc]: Line 40 should be changed to:
**The standard group filter cannot pass to the new page**

File: global_hub/global_hub_known_issues.adoc [hunk 6]
```diff
@@ -21,10 +21,43 @@ Known issue process and when to write:
...
+
+In the *Global Hub Policy Group Compliancy Overview* hub dashboards, you can check one data point by clicking **View Offending Policies for standard group**, but after you click this link to go to the offending page, the standard group filter cannot pass to the new page. 
+
+This is also an issue for the **Cluster Group Compliancy Overview**.
+
+[#detach-managed-hub-deletions]
+== Detach managed hub cluster causes applications and policies deletions
+
+If Multicluster Global Hub Operator is running on {product-title-short} 2.7.9 when the managed hub cluster is detached, all policies and applications that are created on the managed hub cluster are deleted.
```
Comment (ldpliu) [inline global_hub/global_hub_known_issues.adoc]: Line 49 should be :
If Multicluster Global Hub Operator is running on {product-title-short} 2.7.9 **or earlier** when the managed hub cluster is detached, all policies and applications that are created on the managed hub cluster are deleted.

File: global_hub/global_hub_known_issues.adoc [hunk 7]
```diff
@@ -21,10 +21,43 @@ Known issue process and when to write:
...
 
 
-[#global-hub-issues]
-== Multicluster global hub
+[#managed-cluster-not-counted]
+== Managed cluster displays but is not counted
 
-Multicluster global hub known issues and limitations are part of the multicluster global hub documentation.
\ No newline at end of file
+A managed cluster that is not created successfully, meaning `clusterclaim id.k8s.io`` does not exist in the managed cluster, is not counted in the policy compliance dashboards, but shows in the policy console.
```
Comment (cadawson) [inline global_hub/global_hub_known_issues.adoc]:
```suggestion
A managed cluster that is not created successfully, meaning `clusterclaim id.k8s.io` does not exist in the managed cluster, is not counted in the policy compliance dashboards, but shows in the policy console. 
```

**Review by cadawson — COMMENTED** [review]
Just noticed a typo that you would have noticed later.

File: global_hub/global_hub_known_issues.adoc (hunk 7 above)
Comment (swopebe) [reply global_hub/global_hub_known_issues.adoc]: might have noticed lol --thanks for catching @cadawson

File: global_hub/global_hub_known_issues.adoc (hunk 6 above)
Comment (swopebe) [reply global_hub/global_hub_known_issues.adoc]: @ldpliu and @cadawson one final review/approval please?

**Comment by ldpliu** [comment]
It looks good to me.

**Pull request closed without merging**